lib/vmfloaty.rb

```diff
@@ -562,6 +562,25 @@ def run
       end
     end
 
+    command :completion do |c|
+      c.syntax = 'floaty completion [options]'
+      c.summary = 'Outputs path to completion script'
+      c.description = "Outputs path to a completion script for the specified shell, or bash if not specified.\nWill exit non-zero if no completion script is available for the requested shell."
```

Comment:
I think we should include the help text from the readme here just so people understand how to use this command (i.e. sourcing the file this command outputs) just so when people do `floaty completion --help` they have some idea what to do with the command.